.github/workflows/prepare-release.yml

```diff
@@ -19,6 +19,32 @@ on:
         required: true
         description: Current version (beta, like 0.69.1). Don't include `v`.
 jobs:
+  #validate-version format
+  validate-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate version format
+        run: |
+          if [[ ! "${{ inputs.candidate-stable }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+-rcv[0-9]+$ ]]; then
```

Comment:
I dont think we'll be doing `rc` version anymore.  I think we should be enforcing stable version format as 3 sections, starting with a number >= 1.